server/metrics/measurement.go

```diff
@@ -91,6 +93,10 @@ func (m *Measurement) IncrementCount(scope tally.Scope, tags map[string]string,
 	scope.Tagged(tags).Counter(counterName).Inc(count)
 }
 
+func (m *Measurement) SetNDocs(value int64) {
+	m.nDocs += value
```

Comment:
is it thread safe?

Reply:
Yes. A measurement is created at the beginning of the request and the scope of one measurement struct is the request.